### PR TITLE
Resolved Issue #533, Issue #534 and Issue #535 

### LINKED ISSUE #533: Group #16 - Placement_cell Module - Error in Placement_statistics (Issue #2)
AttributeError at /placement/statistics/ =  'method_descriptor' object has no attribute 'today'.

### LINKED ISSUE #534: Group #16 - Placement_cell Module - Error in Manage_records (Issue #3)
AttributeError at /placement/manage_records/= 'method_descriptor' object has no attribute 'today'

### LINKED ISSUE #535: Group #16 - Placement_cell Module - Error in Delete_placement_statistics (Issue #4)
AttributeError at /placement/statistics/ = 'method_descriptor' object has no attribute 'today'.
(As there is error in placement_statistic function, the same error is in delete_placement_statistics as well)

### PR DESCRIPTION
**Pull request by geetikn**
The Attribute Error at /placement/statistics/ and /placement/manage_records/ has been resolved.
Issue #533 #534 and #535

### CODE REVIEW

File: FusionIIIT/applications/placement_cell/forms.py [hunk 1]
```diff
@@ -484,7 +484,8 @@ def clean_company_name(self):
         return company_name
 
 def current_year():
-    return datetime.date.today().year
+    # return datetime.date.today().year --- Issue # 2, 3 & 4
```
Comment (MananJethwani) [inline FusionIIIT/applications/placement_cell/forms.py:487]: please remove the comments

**Review by MananJethwani — CHANGES_REQUESTED** [review]

File: FusionIIIT/applications/placement_cell/forms.py (hunk 1 above)
Comment (chiragagrawal026) [inline FusionIIIT/applications/placement_cell/forms.py]: Please remove this comment.

Comment (chiragagrawal026) [reply FusionIIIT/applications/placement_cell/forms.py]: Otherwise, the changes look fine to me.

**Review by chiragagrawal026 — APPROVED** [review]
@MananJethwani, I have reviewed the pull request and the changes seem alright to me.
Please see to it and merge if correct.

**Pull request merged**